packages/create-pantheon-decoupled-kit/src/templates/next-wp-search-api-addon/pages/search/searchPage.module.css.ts

```diff
@@ -0,0 +1,6 @@
+import { TemplateFn } from '@cli/types';
+import { searchPageCSSModules } from '@partials/nextjs-shared/searchPageCSSModules';
+
+const css: TemplateFn = () => `${searchPageCSSModules()}`;
```

Comment:
```suggestion
const css: TemplateFn = () => searchPageCSSModules();
```

The `${}` is somewhat redundant here.